data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1918 @@
+		event "resistance detained rb"
+		"assisted heliarch" ++
+		payment 1320000
+		conversation
+			`Though you never managed to find out if there were any hostages on board, the Heliarchs stationed at <planet> still herald you as a hero for "detaining the bandits". They immediately set to repair your ship. You're still on alert for a few hours in case any more ships come through, but nothing appears to be out of the ordinary. Near the end of the day, the station returns to normal as well; communications and other functions are restored.`
+			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account, and you're put on a call with the consul.`
+			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and, is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons, letting out what you can't quite put to be a gasp or a roar, only knowing your ears don't want to hear it again.`
+			`	"Bring this report to the other consuls, at once I will. Further tasks, of you we may ask, but for now, done your work is, Captain," he ends the transmission after a short salute. Heliarch ships rush back in as the local garrison returns from Remote Blue. Many of the ship captains come to personally congratulate you in stopping the fleeing ships.`
+
+event "resistance fleeing rb"
+	system "12 Autumn Above"
+		remove fleet "Heliarch"
+
+event "resistance detained rb"
+	system "12 Autumn Above"
+		add fleet "Heliarch" 800
+
+
+
+mission "Heliarch Drills 1"
+	name "Refuelling Drill"
+	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."
+	minor
+	source
+		near "Belug" 6 100
+	destination "Belug's Plunge"
+	to offer
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+		random < 8
+		"combat rating" >= 8104
+	on offer
+		conversation
+			`A pair of Heliarch Judicators land by your ship as you leave for the spaceport. The workers storm to the ships, getting ready to refuel them. Coming down from one of the Judicators, a young Saryd with black hair barely reaching her shoulders pulls out a watch of sorts, seeming to time how long they take to completely fill up the ships' fuel tanks.`
+			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part, it is, in testing how well prepared for emergencies we are."`
```

Comment:
```suggestion
			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part it is, in testing how well prepared for emergencies we are."`
```